src/machine/board_reelboard.go

```diff
@@ -47,3 +47,13 @@ const (
 	SPI0_MOSI_PIN Pin = 45
 	SPI0_MISO_PIN Pin = 46
 )
+
+// PowerSupplyActive enables the supply voltages for nRF52840 and peripherals (true) or only for nRF52840 (false)
```

Comment:
Might want to add to this comment that "This controls the TPS610981 boost converter. You must turn the power supply active in order to use the EPD and other onboard peripherals."